createBenchmark.sh

```diff
@@ -44,7 +44,7 @@ set -x
 
 mvn clean install -DskipTests -DskipChecks -T1C
 
-docker build --build-arg DISTBALL=dist/target/camunda-zeebe-*.tar.gz --build-arg APP_ENV=dev -t "gcr.io/zeebe-io/zeebe:$benchmark" .
+docker build --build-arg DISTBALL=dist/target/camunda-zeebe-*.tar.gz -t "gcr.io/zeebe-io/zeebe:$benchmark" .
```

Comment:
In discarded the idea to just retag the current SNAPSHOT image here and as building an image here allows to create benchmarks for arbitrary branches which we would loose otherwise.